packages/core/src/external/athenahealth/index.ts

```diff
@@ -585,6 +597,7 @@ class AthenaHealthApi {
         await this.subscribeToEvent({
           cxId,
           feedtype: "appointments",
+          eventType: "ScheduleAppointment",
```

Comment:
This change looks good.

But looking at the code, it's not obvious why we're doing this on 403. Following the practice of documenting the "whys", can we add some comment explaining why we subscribe to a schedule appt when we get a 403 from Athena, please?

Reply:
Done.